narwhals/dataframe.py

```diff
@@ -271,6 +282,8 @@ def to_pandas(self) -> Any:
         Convert this DataFrame to a pandas DataFrame.
 
         Examples:
+            Construct pandas and polars DataFrames:
```

Comment:
```suggestion
            Construct pandas and Polars DataFrames:
```
(technically pandas should always be lowercase)

Reply:
Fixed